File: Python/card.py
from payment import Payment

class Card(Payment):
    idCard = int
    number = int
    cvv = int
    date = str

    def __init__(self, id, idCard, number, cvv, date):
        super.__init__(id)
        self.idCard = idCard
        self.number = number
        self.cvv = cvv
        self.date = date

File: Java/Card.java
public class Card extends Payment{
    public Integer idCard;
    public Integer number;
    public Integer cvv;
    public String date;

    public Card(Integer id, Integer idCard, Integer number, Integer cvv, String date){
        super(id);
        this.idCard = idCard;
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}
